requirements.txt

```diff
@@ -25,6 +25,15 @@ onnxscript==0.1.0.dev20241018
 monai==1.4.0
 gif==23.3.0
 scikit-image==0.24.0
+py-spy==0.4.0
 typer==0.12.5
 transformers==4.45.2
 torch-geometric==2.6.1
+timm==1.0.11
+tqdm==4.67.0 
+evaluate==0.4.3
+h5netcdf==1.4.0
+wget==3.2
+pyyaml==6.0.2
+torch==2.5.1
```

Comment:
Please do not add torch to the `requirements.txt` file, its version is managed in the `env.yaml` file (and in the `Dockerfile`).